modin/pandas/test/test_general.py

```diff
@@ -295,15 +295,15 @@ def test_merge_asof_bad_arguments():
     modin_left, modin_right = pd.DataFrame(left), pd.DataFrame(right)
 
     # Can't mix by with left_by/right_by
-    with pytest.raises(ValueError), warns_that_defaulting_to_pandas():
+    with pytest.raises(ValueError):
```

Comment:
this was trying to catch default-to-pandas warnings of a pure pandas code (impossible)